index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="container">
+        <div id="template1">
+            <div class="bar">
+                <p>9:41</p>
+                <img class="mobile_network" src="images/header_img/tower.png" alt="network_img">
+                <img class="mobile_wifi" src="images/header_img/Wifi.png" alt="wifi_logo">
+                <img class="mobile_battery" src="images/header_img/Battery.png" alt="battery">
+            </div>
+            <header>
+                <a href="#" class="logo">My Logo</a>
+                <nav class="ham">
+                <img src="images/header_img/Union.png" alt="bar">
+                </nav>
+            </header>
+            <section>
+                <div class="bg_image">
+                    <img class="background" src="images/header_img/header.png" alt="background_img">
+                </div>
+                <div class="intro">
+                    <h1>I’m Tushar Glad to see you!</h1>
+                </div>
+                <div class="description">
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+                <div class="connect">
+                    <h3>LET'S CONNECT</h3>
+                </div>
+                <ul>
+                    <li><a href=""><img class="twitter" src="images/header_img//twiter.png" alt="twiter_logo"></a></li>
+                    <li><a href=""><img class="linkedIn" src="images/header_img//linkedin.png" alt="linkedIn_logo"></a></li>
+                    <li><a href=""><img class="m" src="images/header_img//m.png" alt="m_logo"></a></li>
+                    <li><a href=""><img class="github" src="images/header_img//github.png" alt="github_logo"></a></li>
+                    <li><a href=""><img class="ok" src="images/header_img//ok.png" alt="ok_logo"></a></li>
+                </ul>
+            </section>
```

Comment:
- [ ] Please note that the background color of the navbar & the headline section should be both white (`#ffffff`) and not `#f7f7f9` only the body of the website is using that color , kindly fix that